api/prisma/schema.prisma

```diff
@@ -21,8 +21,6 @@ model CoreUnit {
 
 model User {
   id       String @id @default(uuid())
```

Comment:
The primary identity of the user should be the eth address in the format `eth:0x...`